playbooks/roles/tarsnap/defaults/main.yml

```diff
@@ -1,12 +1,56 @@
 ---
 
+# Role Internal variables
```

Comment:
@taranjeet Some formatting nits: Could you please

* insert a blank line below this one.
* indicate that this is a "section heading" by adding two more `#` chars. The result should look like this:
    ```
    ### Role-internal variables
    ```